eclipse-theia/theia
# working notebook cell language select

#### What it does

This allows clicking the language on the bottom right of notebook cells and change the cell language. 
For this it moves some of the editor-command code to select a code language to its own service

Also fixes a small but pretty substantial problem with notebook cells and not being able to create new lines when pressing enter

#### How to test

Open a notebook. 
Create a multi line cell check that enter works correctly
Click on the language tag in the lower right corner of the cell and change the language

#### Follow-ups



#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
